oddbird/oddvest
# Upgrade Node LTS and package Yarn with the repo.

## Cute animal pic
_Because everyone likes pictures of animals._

![chinchilla](https://s-media-cache-ak0.pinimg.com/originals/14/4b/4c/144b4cb40239e9bfb2bbcac91427c88b.jpg)

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

- N/A

## Description
_The commit messages say what you did; this should explain why and how._

- Node LTS is now 12.13.0 -- upgraded to that version.
- We use nvm for node version management in local dev, but don't have a great solution for yarn version management. After exploring yvm (https://yvm.js.org/docs/overview), I decided to use the official yarn recommendation, which bundles yarn in the repo itself. See: https://yarnpkg.com/en/docs/cli/policies

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally._

- Run `nvm install` to upgrade to Node LTS. Output should end with `Now using node v12.13.0`
- Run `yarn` to install front end dependencies. Output should start with `yarn install v1.19.1`

## Impacted areas of website

- Local dev environment

_Provide URLs and/or a description of where to look._

- N/A

## Show me
_Provide screenshots/animated gifs/videos if necessary._

- N/A

---

## Reviewer tasks:
